src/push-notifications.test.js

```diff
@@ -126,41 +127,40 @@ describe('interest methods', () => {
 
       dorequest.default = mockDoRequest;
 
-      return PusherPushNotifications.init({
+      const beamsClient = new PusherPushNotifications.Client({
         instanceId,
-      })
-        .then(beamsClient => beamsClient.addDeviceInterest(interest))
-        .then(() => {
-          expect(mockDoRequest.mock.calls.length).toBe(1);
-          expect(mockDoRequest.mock.calls[0].length).toBe(1);
-          expect(mockDoRequest.mock.calls[0][0]).toEqual({
-            method: 'POST',
-            path: [
-              'https://df3c1965-e870-4bd6-8d75-fea56b26335f.pushnotifications.pusher.com',
-              '/device_api/v1/instances/df3c1965-e870-4bd6-8d75-fea56b26335f',
-              '/devices/web/web-1db66b8a-f51f-49de-b225-72591535c855',
-              '/interests/donuts',
-            ].join(''),
-          });
+      });
+      beamsClient.addDeviceInterest(interest).then(() => {
```

Comment:
Should we be returning the promise here, or doesn't it matter? (There are a number of other places this change would need to be made too)